core/vm/analysis.go

```diff
@@ -26,6 +26,14 @@ func codeBitmap(code []byte) []uint64 {
 	for pc := 0; pc < len(code); {
 		op := OpCode(code[pc])
 		pc++
+		// Short circruit for now on EOF ops with immediates.
+		// TODO(matt): make EOF-specific code bitmap
+		if op == RJUMP || op == RJUMPI || op == CALLF {
+			// TODO CZ: verify that this is correct
```

Comment:
Fixed the code with https://github.com/ETCCooperative/erigon/pull/4/commits/4ea3d3c1ba09c041a1199d409039d7877d493499. 

I think it might be able to receive some optimisations

Reply:
Re-fixed the code so as all tests are passing as well with https://github.com/ETCCooperative/erigon/pull/4/commits/513d63f1ee334e93d6235c92828f0b4e6c479f87.

This also applies @holiman's suggestion https://github.com/ethereum/go-ethereum/pull/26133/files#r1026320917